cmd/eksctl-anywhere/cmd/upgrademanagementcomponents.go

```diff
@@ -85,7 +96,7 @@ var upgradeManagementComponentsCmd = &cobra.Command{
 		}
 
 		validator := management.NewUMCValidator(managementCluster, deps.Kubectl)
-		return runner.Run(ctx, clusterSpec, managementCluster, validator)
+		return runner.Run(ctx, bundles, eksaRelease, clusterSpec, managementCluster, validator)
```

Comment:
I would suggest maybe grouping bundles and release in one struct. the method signature is getting quite big
Like `cluster.Release`?